db/upgrade.php

```diff
@@ -163,5 +163,35 @@ function xmldb_plagiarism_unicheck_upgrade($oldversion) {
         upgrade_plugin_savepoint(true, 2018021531, 'plagiarism', 'unicheck');
     }
 
+    if ($CFG->branch >= 39) {
```

Comment:
Remove all updates to the current point. We don't need updates from previous versions